index.js

```diff
@@ -1,18 +1,67 @@
 const readline = require('readline');
 const rl = readline.createInterface(process.stdin, process.stdout);
 
+// functions are here
+
 function ask(questionText) {
   return new Promise((resolve, reject) => {
     rl.question(questionText, resolve);
   });
 }
 
+function binarySearch(min, max) {
+  return Math.floor((max + min) / 2);
+
+}
 start();
 
 async function start() {
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
-  console.log('You entered: ' + secretNumber);
-  // Now try and complete the program.
+  console.log("Let's play a game where you (the human) think of a number between 1 and n and I (the computer) try to guess it.")
+
+  await ask("Press ENTER when you're ready. ");
+
+  // variables within the async function are here
+
+  let b = parseInt(await ask("Great. Please give me the upper range for my guesses. "));
+  
+  // I tried to set the default here, but it doesn't work
+  // if (b !== Number) {
+  //   b = 100;
+  // }
```

Comment:
There are couple ways that you could get this to work. One way:
```js
if ( typeof b !== 'number' ) {
  b = 100;
}
```
Another way would be:
```js
if ( isNaN(b) ) {
  b = 100
}
```